tests/testGroupStudent.js

```diff
@@ -0,0 +1,657 @@
+const ERR = require('async-stacktrace');
+
+const config = require('../lib/config');
+const path = require('path');
+var assert = require('chai').assert;
+var request = require('request');
+var cheerio = require('cheerio');
+
+
+
+var sqldb = require('@prairielearn/prairielib/sql-db');
+var sqlLoader = require('@prairielearn/prairielib/sql-loader');
+var sql = sqlLoader.loadSqlEquiv(__filename);
+
+var helperServer = require('./helperServer');
+
+// cd /PrairieLearn
+// docker/start_postgres.sh
+// git config --global user.email "dev@illinois.edu"
+// git config --global user.name "Dev User"
+// npm test -- --grep "Group based homework assess control on student side"
+
+let res, page, elemList;
```

Comment:
```suggestion
let page, elemList;
```

I'm getting a linter warning that `res` is never used